ide/static/js/content.js

```diff
@@ -251,8 +269,8 @@ class Content extends React.Component {
     Object.keys(positions).forEach(layerId => {
       const layer = net[layerId];
       layer.state = {
-        top: `${250 + 50 * positions[layerId][1]}px`,
-        left: `${20 + 180 * positions[layerId][0]}px`,
+        top: `${height + 65 * positions[layerId][1]}px`,
```

Comment:
We can ignore this issue (centering fails on some large screens) for now, I guess. Please add a comment here documenting it.